server/shared/userSyncer.go

```diff
@@ -86,7 +89,9 @@ func (s *Syncer) Sync(ctx context.Context, user *model.User, flatPermissions boo
 			} else {
 				remotePerm, err := s.Remote.Perm(ctx, user, repo.Owner, repo.Name)
 				if err != nil {
-					return fmt.Errorf("could not fetch permission of repo '%s': %v", repo.FullName, err)
+					log.Debug().Msgf("could not fetch permission of repo '%s': %v", repo.FullName, err)
+					hasErrors = true
+					continue
```

Comment:
As this code solves a somehow "different" problem, I would suggest to just log the error as debug and ignore it otherwise. So we just keep these 3 lines of code for now.

```suggestion
          // TODO: report errors again after #648 got solved
					log.Debug().Msgf("could not fetch permission of repo '%s': %v", repo.FullName, err)
					continue
```